lib/dogma/rules.ex

```diff
@@ -1,25 +1,23 @@
 defmodule Dogma.Rules do
-
-  to_rule_name = fn path ->
-    name = path
-            |> Path.basename(".ex")
-            |> Mix.Utils.camelize
-            |> String.to_atom
-    Module.concat( Dogma.Rules, name )
-  end
-
-  files = Path.wildcard( "lib/dogma/rules/*.ex" )
-  rules = Enum.map( files, to_rule_name )
-
-  # FIXME: This doesn't work. Removing a file doesn't trigger recompile.
-  for file <- files do
-    @external_resource file
-  end
+  alias Dogma.Rules
 
   @doc """
-  Returns a list of all modules that define a Dogma rule.
+  Returns a list of all modules that define a Dogma rule and the congiguration
+  that will be passed to each rule.
   """
   def list do
-    unquote(rules)
+    [
+      {Rules.DebuggerStatement},
+      {Rules.FinalNewline},
+      {Rules.LineLength, max_length: 80},
+      {Rules.LiteralInCondition},
+      {Rules.ModuleName},
+      {Rules.NegatedIfUnless},
+      {Rules.QuotesInString},
+      {Rules.TrailingBlankLines},
+      {Rules.TrailingWhitespace},
+      {Rules.UnlessElse},
+      {Rules.WindowsLineEndings}
```

Comment:
I don't see any reason why we can't have people namespace their rules under `Dogma.Rules`. Or we could make some mechanism for adding rules in another namespace later.

Either way, let's try and write this as nicely as possible for now. I'm very willing to make breaking changes to the API this early on.